# Supplementary Data for "_Plastid-localized xanthorhodopsin increases diatom biomass and ecosystem productivity in iron-limited surface oceans_"
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8322266.svg)](https://doi.org/10.5281/zenodo.8322266)

The repository contains supplementary data and metadata used in the _Nature Microbiology_ research paper [Plastid-localized xanthorhodopsin increases diatom biomass and ecosystem productivity in iron-limited surface oceans](https://www.nature.com/nmicrobiol/) reporting on a xanthorhodopsin variant [FcR1](https://www.uniprot.org/uniprotkb/A0A1E7EXA4/) isolated from the polar diatom _Fragilariopsis cylindrus_.

## AlphaFold2 3D structure of *Fc*R1
A predicted (3D) structure of FcR1 was generated using [ColabFold](https://github.com/sokrypton/ColabFold). The [Protein Data Bank (PDB)](http://rcsb.org) was scanned for homologous structures using a [Dali search](http://ekhidna2.biocenter.helsinki.fi/dali). The experimental structure of PDB entry [4HYJ](https://doi.org/10.2210/pdb4HYJ/pdb), a top five search hit was manually chosen based on highest quality according to PDB indicators and used for structural alignment with FcR1 using [ChimeraX](https://www.cgl.ucsf.edu/chimerax/) to position the retinal in the predicted FcR1 structure. The final FcR1 model was generated by removing the structure of PDB ID [4HYJ](https://doi.org/10.2210/pdb4HYJ/pdb) and visualized using [ChimeraX](https://www.cgl.ucsf.edu/chimerax/).

A [secondary structure of *Fc*R1](http://wlab.ethz.ch/protter/#up=A0A1E7EXA4&nterm=phobius.nterm&tm=PHOBIUS.TM,255-272&mc=whitesmoke&lc=blue&tml=none&numbers&legend&n:retinal%20Schiff%20base%20K-261,cc:white,fc:deeppink,bc:deeppink=261&n:proton%20donor%20glutamic%20acid%20E-132,cc:white,fc:black,bc:black=132&n:proton%20acceptor%20aspartic%20acid%20D-121,cc:white,fc:black,bc:black=121&n:spectral%20tuning%20Lys-129,cc:white,fc:limegreen,bc:limegreen=129&n:retinal%20binding%20pocket,cc:white,fc:dimgray,bc:dimgray=119,122,126,160,161,165,181,184,185,188,226,229,230,233,253,260&n:Lysine-rich%20motif,s:diamond,cc:white,fc:orange,bc:orange=21-30&n:potential%20signal%20peptide,cc:white,fc:red,bc:red=1-14&format=svg) with key residues highlighted was generated using [Protter v1.0](https://github.com/ulo/Protter) and manually edited.

## Phylogenetic tree of eukaryotic xanthorhodopsins
Rhodopsin sequences focusing on eukaryotic xanthorhodopsin protein sequences were collected from the scientific literature and supplemented with manually curated sequences from BlastP search results of the Fragilariopsis cylindrus Rhodopsin 1 (FcR1) protein sequence [(UniProt: A0A1E7EXA4)](https://www.uniprot.org/uniprotkb/A0A1E7EXA4/) against NCBI databases, a custom database from [Beja Lab](https://github.com/BejaLab), as well as similar sequences at an amino acid sequence identity threshold of ≥50% based on their membership in the [UniRef50 cluster](https://www.uniprot.org/uniref/) built using FcR1 as seed sequence [(UniRef50_A0A7S2RIV1)](https://www.uniprot.org/uniref/UniRef50_A0A7S2RIV1). The selected set of proteins was further supplemented with a manually curated set of outgroup sequences including characterized members of the proteorhodopsin subfamily totalling 242 sequences.

Sequences were aligned using [MAFFT](https://mafft.cbrc.jp/alignment/software/) with default parameters and positions with 25% or more gaps were masked. The maximum-likelihood (ML) reference reconstruction was built using [RAxML v.8.277](https://github.com/stamatak/standard-RAxML) under the LG+G4 model (selected using [IQ-TREE](https://github.com/Cibiv/IQ-TREE) with the corrected Akaike information criterion).

## Diatom growth data at different silicate concentrations
The polar diatom _F. cylindrus_ was grown at different concentrations of silicate ranging from 0.01 - 100 µM to determine mean growth rates and its half-saturation constant _K<sub>m</sub>_ for silicate.

## Expression vectors used in the study
| Vector | Description | Reference |
| ---- | ---- | ---- |
| pGEMHE | _Xenopus laevis_ expression vector pGEMHE containing 5’ and 3’UTRs from a Xenopus beta-globin gene | [Kreig, P. A., and Melton, D.A. (1984), Nucl. Acids Res. 72, 7057-70](https://doi.org/10.1093/nar/12.18.7057); [Liman, E. R. et al. (1992), Neuron 9(5): 861-71](https://doi.org/10.1016/0896-6273(92)90239-A) |
| pPha-T1 | _Phaeodactylum tricornutum_ transformation vector containing fucoxanthin chlorophyll-binding protein (fcp) regulatory sequences to drive constitutive expression of bleomycin resistance protein (Ble) conferring zeocin resistance and gene of interest | GenBank:[AF219942](https://www.ncbi.nlm.nih.gov/nuccore/AF219942); [Zaslavskaia et al. (2001), J. Phycol. 36(2): 379](https://doi.org/10.1046/j.1529-8817.2000.99164.x)|
| StuI-GFP-pPha-T1 | _Phaeodactylum tricornutum_ transformation vector for generation of eGFP fusion proteins. To generate GFP fusion constructs, the sequence of interest is to be cloned into the StuI restriction site at 5’ end of eGFP. Derivate of the pPha-T1 vector | [Gruber et al. (2007), Plant Mol. Biol. 64(5): 519](https://doi.org/10.1007/s11103-007-9171-x); [Zaslavskaia et al. (2001), J. Phycol. 36(2): 379](https://doi.org/10.1046/j.1529-8817.2000.99164.x) |
| pTpFcp-GFP/FcpNat | _Thalassiosira pseudonana_ transformation vector for constitutive expression of eGFP fusion proteins | [Poulsen et al. (2006), J. Phycol. 42(5):1059](https://doi.org/10.1111/j.1529-8817.2006.00269.x); [Scheffel et al. (2011), PNAS 108(8):3175](https://doi.org/10.1073/pnas.1012842108) |

## Files in the repository
+ `data/FcR1_A0A1E7EXA4.cif` - AlphaFold2 structure prediction of FcR1 with retinal in crystallographic information file (CIF) format
+ `data/FcR1_JGI_ID_287459.fasta` - Nucleotide coding sequence (CDS) of _fcr1_ gene (JGI ID: 287459) in fasta format
+ `data/Xanthorhodopsins_Strauss_et_al2023_AlignmentWithGaps.fasta` - raw alignment
+ `data/Xanthorhodopsins_Strauss_et_al2023_MaskedAlignment.fasta` - masked alignment
+ `data/Xanthorhodopsins_Strauss_et_al2023_RAxML.tree` - Newick tree file
+ `data/Strauss_SI_Fig1_SourceData.csv`- Growth rates of _F. cylindrus_ at different silicate concentrations
+ `data/pGEMHE.gb`- Genbank flat file for _Xenopus laevis_ oocyte expression vector pGEMHE
+ `data/pPha-T1.gb`- Genbank flat file for _Phaeodactylum tricornutum_ transformation vector pPha-T1
+ `data/StuI-GFP-pPha-T1.gb`- Genbank flat file for _Phaeodactylum tricornutum_ transformation vector StuI-GFP-pPha-T1
+ `data/pTpFcpGFP-FcpNat.gb`- Genbank flat file for _Thalassiosira pseudonana_ transformation vector pTpFCP-GFP/fcpNat
+ `metadata/Xanthorhodopsins_Strauss_et_al2023_metadata.csv` - metadata file containing xanthorhodopsin sequence data

## Metadata structure
| Column | Description |
| --- | --- |
| Name | Sequence ID/accession |
| Accession | NCBI  accession version |
| Description | Sequence/NCBI definition line |
| Organism | Organism |
| Sequence | Amino acid sequence |
| Taxonomy | Taxonomic lineage |
| db_xref | NCBI taxonomy ID |
| protein_family | Rhodopsin clade/family |
| taxon_group | Taxonomic group |
